code/game/jobs/job/supervisor.dm

```diff
@@ -150,16 +150,18 @@ GLOBAL_DATUM_INIT(captain_announcement, /datum/announcement/minor, new(do_newsca
 /datum/job/blueshield
 	title = "Blueshield"
 	flag = JOB_BLUESHIELD
-	department_flag = JOBCAT_KARMA
+	department_flag = JOBCAT_ENGSEC
 	total_positions = 1
 	spawn_positions = 1
 	supervisors = "the Nanotrasen representative"
 	department_head = list("Captain")
 	selection_color = "#ddddff"
-	req_admin_notify = 1
-	is_command = 1
+	req_admin_notify = TRUE
+	is_command = TRUE
 	transfer_allowed = FALSE
 	minimal_player_age = 21
+	exp_requirements = 3000 // 50 hours baby
```

Comment:
`HOURS` does not work here. This is a minutes value, not conversion of ticks to hours.